tools/topology/topology2/cavs-rt5682-rt1019.conf

```diff
@@ -0,0 +1,283 @@
+<include/common/vendor-token.conf>
+<include/common/tokens.conf>
+<include/common/manifest.conf>
+<include/common/pcm.conf>
+<include/common/pcm_caps.conf>
+<include/common/fe_dai.conf>
+<include/common/route.conf>
+<include/components/host.conf>
+<include/components/dai.conf>
+<include/components/pipeline.conf>
+<include/components/copier.conf>
+<include/dais/ssp.conf>
+<include/dais/dmic.conf>
+<include/dais/pdm_config.conf>
+<include/common/data.conf>
+<include/dais/hw_config.conf>
+<include/pipelines/cavs/passthrough-capture.conf>
+<include/pipelines/volume-playback.conf>
+<include/pipelines/volume-capture.conf>
+
+#
+# List of all DAIs
+#
+# PCM0 ----> volume -----> SSP1  (Speaker - ALC1019)
+# PCM1 <---> volume <----> SSP0  (Headset - ALC5682)
```

Comment:
why did you invert PCM and SSPs

Why not use PCM0 for SSP0 and PCM1 for SSP1.

Simple.always.wins

Reply:
Based on previous topologies for JSL and TGL, SSP1 is used for the SPEAKER and SSP0 for headset
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-jsl-rt5682.m4#L26-L27
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-jsl-rt5682.m4#L168
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-tgl-max98357a-rt5682.m4#L77-L80
https://github.com/thesofproject/sof/blob/main/tools/topology/topology1/sof-tgl-max98373-rt5682.m4#L27-L32